src/applications/disability-benefits/all-claims/pages/paymentInformation.js

```diff
@@ -9,45 +11,64 @@ const {
 } = fullSchema.properties;
 
 export const uiSchema = {
-  'ui:title': 'Payment information',
-  'ui:description':
-    'This is the bank account information we have on file for you. We’ll pay your disability benefit to this account.',
-  bankAccountType: {
-    'ui:title': 'Account type',
+  'view:bankAccount': {
+    'ui:title': 'Payment Information',
+    'ui:description':
+      'We’re currently paying your compensation to this account',
+    'ui:field': ReviewCardField,
     'ui:options': {
-      widgetClassNames: 'va-select-medium-large',
+      viewComponent: PaymentView,
+      reviewTitle: 'Payment information',
+      editTitle: 'Add new bank account',
+      itemName: 'account',
+      startInEdit: formData =>
+        Object.keys(formData).every(key => !formData[key]),
```

Comment:
This seems like it will only start in edit mode if none of the fields are filled, is that the intent? What about incomplete (invalid) account info coming from prefill?

Reply:
In that case, it'll start in edit mode because of the logic that was already in the `ReviewCardField`.